app/code/Magento/Sales/view/adminhtml/templates/order/creditmemo/create/items.phtml

```diff
@@ -126,7 +126,8 @@ $commentText = $block->getCreditmemo()->getCommentText();
                            class="admin__control-checkbox"
                            name="creditmemo[send_email]"
                            value="1"
-                           type="checkbox" />
+                           type="checkbox"
+                           checked />
```

Comment:
Could you please explain why it's checked by default? It's definitely backward incompatible change since the default behavior is changed

Reply:
Sure, It was something that @ihor-sviziev suggested, however since then, with more thought, it is not so clear whether this is the right behaviour. Like you say, it changes the default behaviour.

At the moment, the checkbox is only visible if the global setting to enable emails is set. So, if it is not set globally, the checkbox is hidden, and the email doesn't get sent. If the global setting is enabled, then the checkbox is visible and the user can choose whether to send the email or not.

More than happy to revert that change back though and leave the default to unchecked, so the admin has to have global setting enabled, and also tick the box on every invoice/creditmemo/shipment to send the email. If the global setting is enabled, and the box is left unchecked, then the email doesn't get sent. This seems the least contentious solution.

Reply:
Hi @gwharton ,
I just analyzed all use cases (thx for good and detailed explanation) - seems like we should not check the checkbox by default. Such approach will be more clear and backward compatible. 
Could you revert this change? Sorry for confusing you last time